cmd/device-hub/cmd_server.go

```diff
@@ -23,6 +23,13 @@ import (
 var serverCommand = &cobra.Command{
 	Use:   "server",
 	Short: "Start device hub.",
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		// Load Config File if is enabled
+		if _config.ConfigFile {
+			_config.AddConfigFile(_config.ConfigPath)
+
+		}
+	},
```

Comment:
Could you add the `PersistentPreRun` to the root cammand and see if that means that it is run on every child command?  Thanks